Cargo.toml

```diff
@@ -448,15 +455,78 @@ members = [
 	"substrate/utils/wasm-builder",
 ]
 
-[workspace.package]
-authors = ["Parity Technologies <admin@parity.io>"]
-edition = "2021"
-repository = "https://github.com/paritytech/polkadot-sdk.git"
-license = "GPL-3.0-only"
-version = "1.0.0"
+[profile.release]
+# Polkadot runtime requires unwinding.
+panic = "unwind"
```

Comment:
I guess this is generally fine?

Reply:
Should probably be the default anyway and was only required for the native runtime